docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_monitoring/_index.md

```diff
@@ -0,0 +1,235 @@
+---
+type: docs
+title: "Azure Monitor"
+linkTitle: "Azure Monitor"
+weight: 12
+description: >
+---
+
+## Enable Azure Monitor on Azure Arc-enabled servers
+
+The scenario will show you how to onboard Azure Arc-enabled servers to [Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/overview), so you can monitor your Linux and Windows servers running on-premises or at other cloud providers.
+
+In this guide, you will create the following Azure resources that support this Azure Monitor scenario:
+
+- Log Analytics workspace.
+
+- Log data sources: performance counters and events for Windows & Linux.
+
+- VM Insights solution.
+
+- Azure Monitor OS sample alerts.
+
+- Azure Workbooks: AlertsConsole, OSPerformanceAndCapacity and WindowsEvents.
+
+- Azure Dashboard: monitoring overview for Azure Arc-enabled servers.
+
+- Azure Policies to automate agents deployment:
```

Comment:
Consider just assigning the "[Enable Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/vm/vminsights-enable-policy)" initiative instead of multiple policies if it makes sense

Reply:
Hi @sebassem! I thought about including it, but this initiative contains 10 policies and just 4 of them apply to Azure Arc-enabled servers. That's why I decided to just assign those four so I don't involve and connect Azure VMs. What do you think?

Reply:
makes sense, let' keep it that way.